tools/sync-npm-deps-to-tsc-projects.ts

```diff
@@ -5,8 +5,9 @@ import JSON5 from "json5";
 import fs from "fs-extra";
 import { globby, Options as GlobbyOptions } from "globby";
 import { RuntimeError } from "run-time-error";
-import lernaJson from "../lerna.json";
+import { readFileSync } from "fs";
 
+const lernaJson = JSON.parse(readFileSync("lerna.json", "utf8"));
```

Comment:
@aldousalvarez Please 
1. Move this logic to within the main function and make sure to

2. use the readFileAsync call instead so that we don't block (it doesn't make much of a difference in a very simple utility script like this but regardless best practices are like muscle memory so I advocate for adhereing to them even when it does not seem crucial to do so)

3. Make sure to parse the JSON file with an absolute path (e.g. leverage the already computed `PROJECT_DIR` variable)

Reply:
@petermetz already done with the latest commit. re-requested to review the changes made. Thank you so much